docker/Makefile

```diff
@@ -33,6 +33,14 @@ lDMax=3000
 lM=0.2
 lMMax=100
 
+UserForwardPayloadLength=2000
```

Comment:
you just now said you don't want all the tests to be the same size. I've changed the payload size of the ecdh sphinx test to be 50000 bytes.

The payload size is available via the geometry object. We don't want to make variables for a default sphinx geometry since the whole point of the ticket is to make it so that the Sphinx geometry is controlled by the dirauths and is present in everyone's configuration. But some clients might want their own variable verify against that geometry object. In that specific context.